.travis.yml

```diff
@@ -1,12 +1,28 @@
+language: python
+sudo: false
+
 env:
    global:
       - image=pythonnode_slave
-language: python
-python:
-  - "2.7"
-env:
-   - TRAVIS_NODE_VERSION="4"
-   - TRAVIS_NODE_VERSION="6"
+
+matrix:
+  include:
+    - python: 2.7
+      env: TRAVIS_NODE_VERSION="4"
+    - python: 2.7
+      env: TRAVIS_NODE_VERSION="6"
+    - python: 3.5
+      env: TRAVIS_NODE_VERSION="4"
+    - python: 3.5
+      env: TRAVIS_NODE_VERSION="6"
```

Comment:
no need to test all the combinations of node4 and node6. just test python 3.5/3.6 with node 6